src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -141,7 +141,7 @@ internal override FlowAnalysisAnnotations FlowAnalysisAnnotations
             }
         }
         internal override bool IsNullChecked
```

Comment:
nit: please put a blank line between the `}`  of `FlowAnalysisAnnotations`'s accessor list and this declaration.